src/components/grid/GridInfoDialog.vue

```diff
@@ -0,0 +1,84 @@
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas at eleifend orci, vitae maximus felis. Aenean justo mauris, ullamcorper sed nulla id, tincidunt semper turpis. Aliquam ultrices elit non vehicula rhoncus. Cras porttitor justo magna, in mollis odio vulputate sit amet. Sed vitae lorem non felis semper vehicula a commodo ex. Aenean id ante eget nunc commodo viverra lobortis vel nulla. Pellentesque vestibulum porttitor neque, sit amet condimentum massa maximus a. Aenean eleifend velit ut arcu ullamcorper elementum. Phasellus elementum eu augue vel malesuada. Vestibulum ut dui sed nulla dictum fermentum in in tellus. Duis porttitor sed turpis in tempor. Pellentesque vitae vestibulum felis. Proin porta, arcu at efficitur hendrerit, neque orci bibendum felis, eu pharetra ipsum enim at mauris. Integer porttitor, diam quis varius egestas, augue diam venenatis turpis, ac auctor nisl dui eu erat.</p>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas at eleifend orci, vitae maximus felis. Aenean justo mauris, ullamcorper sed nulla id, tincidunt semper turpis. Aliquam ultrices elit non vehicula rhoncus. Cras porttitor justo magna, in mollis odio vulputate sit amet. Sed vitae lorem non felis semper vehicula a commodo ex. Aenean id ante eget nunc commodo viverra lobortis vel nulla. Pellentesque vestibulum porttitor neque, sit amet condimentum massa maximus a. Aenean eleifend velit ut arcu ullamcorper elementum. Phasellus elementum eu augue vel malesuada. Vestibulum ut dui sed nulla dictum fermentum in in tellus. Duis porttitor sed turpis in tempor. Pellentesque vitae vestibulum felis. Proin porta, arcu at efficitur hendrerit, neque orci bibendum felis, eu pharetra ipsum enim at mauris. Integer porttitor, diam quis varius egestas, augue diam venenatis turpis, ac auctor nisl dui eu erat.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faTimes } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+
+library.add(faTimes)
+
+export default Vue.extend({
+  name: 'GridInfoDialog',
+  components: { FontAwesomeIcon }
+})
+</script>
+
+<style lang="scss" scoped>
+  svg{
+    cursor: pointer;
+    &:hover path{
+      fill: rgba(255,255,255,0.6);
+    }
+  }
+  .info-overlay{
+    position: fixed;
+    left:0;
+    right:0;
+    top:0;
+    bottom:0;
+    z-index: 1031;
```

Comment:
please add note on why this number is 1031, magic...

Reply:
I told @jelmerveen you would say that. :)

It's still a work in progress and me and @jelmerveen were trying different things / experimenting, we're not going to leave it like this.